src/app/main-menu/main-menu.component.html

```diff
@@ -4,11 +4,54 @@
             <img [src]="path">
         </li>
     </ul>
+    <div class="yellow-line">
+        
+    </div>
     <div class="contact-menu-tel">
         <img src="../../assets/icons/tel.svg">
         <a href="tel:028811888">02-881-1888</a>
     </div>
 </div>
-<div>
-    สีขาว นะจ๊ะ
+
+<div class="product-menu">
+    <ul class="product-menu-list">
+        <li>
+            <img class="logo" src="../../assets/pictures/logo.png">
+        </li>
+        <li class="main-menu">
+            <span class="text-menu">ซื้อประกัน</span>
+            <img src="../../assets/icons/next.svg">
```

Comment:
I think you can use icon bootstrap https://icons.getbootstrap.com/ instead of svg image